src/azul/__init__.py

```diff
@@ -309,6 +309,17 @@ def cart_item_state_machine_name(self):
     def cart_api_ip_whitelist(self):  # TODO: Remove when authentication is added
         return os.environ['AZUL_CART_API_IP_WHITELIST'].split(' ')
 
+    @property
+    def access_token_issuer(self):
+        return "https://humancellatlas.auth0.com/"
+
+    @property
+    def access_token_audience_list(self):
+        return [
+            f"https://{self.deployment_stage}.data.humancellatlas.org/",
+            f"{self.access_token_issuer}userinfo"
```

Comment:
… it makes this look nicer:

```suggestion
            f"{self.access_token_issuer}/userinfo"
```